source/includes/configuration/_02-get-historical-snapshot.md.erb

```diff
@@ -1,12 +1,18 @@
 ## Get configuration
 
 ```shell
-$ curl 'https://ci.example.com/go/api/admin/config/5059cf548db9ea2d1b9192b45529ccf0.xml' \
+$ curl 'https://ci.example.com/go/api/admin/config.xml' \
       -u 'username:password'
 ```
 
 > The above command returns the contents of the config file
 
+
```

Comment:
I see what both of you are saying. You're right. I was wrong. The HTTP headers are in the others too and it makes sense.